### DISCUSSION TITLE
Does onion retroarch gba support IPS autopatching?

### DISCUSSION THREAD

**Discussion by erfanCA01** [OnionUI/Onion#1416] [Q&A] [answered]
I know about manual patched roms, but i am playing pokemon emerald and there are bug fix patches that i want to apply without modifying the base game as i want to play the game without patched fix as well. I know pc retroarch supports auto patching, not sure if it is same process on onionOS

**Comment by thomasklemenc** [comment, marked as answer]
We are compiling RetroArch with `HAVE_PATCH=1` so it _should_ work. Have you tried it?

**Reply by erfanCA01** [reply]
just tried it , yep it works with mgba core, multi patch works too with "rom".ips, "rom",ips1 and so on, thanks!,

is there any way to check which retro arch version i am running on onion os?

**Reply by thomasklemenc** [reply]
Yes - you can open RetroArch, go to Information->System Information and it will show the build version in the bottom (currently 0.15.0).

**Reply by erfanCA01** [reply]
ah so currently 1.15.0, is there any way I can update to 1.17.0 by my self?
